src/options.h

```diff
@@ -416,6 +416,11 @@ sp_after_semi_for_empty;
 extern Option<iarf_e>
 sp_before_square;
 
+// Whether to ignore sp_before_square for 'asm' blocks (i.e. assume they
+// have their own indentation).
+extern Option<bool>
```

Comment:
why not `Option<iarf_e>` ?

Reply:
I saw another option implement something alike in indent_ignore_asm_block implemented as a bool.
Using iarf_e is also possible. And is more like the default behavior of course.

Just tell me what the uncrustify maintainers like most!

Reply:
I like iarf

Reply:
Please use the same comment as the description from options.h

Reply:
Please change the source as proposed at my comment just before.
It is important to have the same text at all the places

Reply:
I think I have the comment "Add or remove space before '[' for asm block." everywhere.
(space.cpp, options.h, mini_nd_ucwd.txt, mini_d_ucwd.txt, universalindent.cfg, show_config.txt and defaults.cfg).